spec/models/form/sales/pages/discounted_ownership_type_spec.rb

```diff
@@ -5,7 +5,26 @@
 
   let(:page_id) { nil }
   let(:page_definition) { nil }
-  let(:subsection) { instance_double(Form::Subsection) }
+  let(:subsection) { instance_double(Form::Subsection, form: instance_double(Form, start_date:)) }
+  let(:start_date) { Time.zone.today }
+
+  describe "headers" do
+    context "when form year is for 2023" do
+      let(:start_date) { Time.utc(2023, 2, 8) }
```

Comment:
I find it slightly odd that these aren't the 1st of April, since that's when form start dates are. But it doesn't actually matter at the moment, and in practice I can see that other dates do float around in our testing so :shrug:

Reply:
It's weird but with things like dates and times I like to test within the boundaries instead of on the first/last day. I guess it's a habit picked up from working with dates that change even though we pretty much know that doesn't happen here.

Reply:
This would actually mean that the form start year would be 2022 (cause 2022 runs 22/04/01 till 23/04/01) so either the test name or the date should change